### ISSUE TITLE
Error in Validator Stats page

### ISSUE THREAD

**Issue by miloskriz** [polkadot-js/apps#9590]
* **I'm submitting a ...**

  - [x] Bug report
  - [ ] Feature request
  - [ ] Support request
  - [ ] Other

* **What is the current behavior and expected behavior?**

When navigating to `Network` -> `Staking` -> `Validator Stats`, the pane titled `rewards & slashes` is throwing an uncaught error:

```
Uncaught error. Something went wrong with the query and rendering of this component. Please supply all the details below when logging an issue, it may help in tracing the cause.

Cannot read properties of undefined (reading 'enabled')

TypeError: Cannot read properties of undefined (reading 'enabled')
at Object.afterDraw (https://cloudflare-ipfs.com/ipns/dotapps.io/modu.2743e2706832723b.js:2:90133)
at p (https://cloudflare-ipfs.com/ipns/dotapps.io/modu.2743e2706832723b.js:2:353822)
at Et._notify (https://cloudflare-ipfs.com/ipns/dotapps.io/modu.2743e2706832723b.js:2:297210)
at Et.notify (https://cloudflare-ipfs.com/ipns/dotapps.io/modu.2743e2706832723b.js:2:296999)
at Gt.notifyPlugins (https://cloudflare-ipfs.com/ipns/dotapps.io/modu.2743e2706832723b.js:2:316334)
at Gt.draw (https://cloudflare-ipfs.com/ipns/dotapps.io/modu.2743e2706832723b.js:2:311743)
at Gt.render (https://cloudflare-ipfs.com/ipns/dotapps.io/modu.2743e2706832723b.js:2:311315)
at Gt.update (https://cloudflare-ipfs.com/ipns/dotapps.io/modu.2743e2706832723b.js:2:309373)
at Gt.<anonymous> (https://cloudflare-ipfs.com/ipns/dotapps.io/modu.2743e2706832723b.js:2:304939)
at Gt._doResize (https://cloudflare-ipfs.com/ipns/dotapps.io/modu.2743e2706832723b.js:2:358229)
```

whereas usually this pane should be populated with a graph.

* **What is the motivation for changing the behavior?**

N/A

* **Please tell us about your environment:**

  - Version: apps v0.131.4
  - Environment: Brave Explorer on Windows 11

    - [ ] Node.js
    - [x] Browser
    - [ ] Other (limited support for other environments)

  - Language:

    - [ ] JavaScript
    - [ ] TypeScript (include tsc --version)
    - [ ] Other

**Comment by jacogr** [comment]
Which network is this? (The reason for asking is that networks have different capabilities in terms of staking, i.e. running different versions and then obviously some stand-alone networks has also made changes to staking. So it is quite important to know on which network this appears)

**Comment by miloskriz** [comment]
Hello @jacogr !

Please note that this behaviour can be seen at least in the following relaychains and stand-alone networks:

- Polkadot (`IPFS` apps v0.131.4 and `polkadot.js.org` apps v0.131.5-40)
- Kusama (`IPFS` apps v0.131.4 and `polkadot.js.org` apps v0.131.5-40)
- Westend (`IPFS` apps v0.131.4 and `polkadot.js.org` apps v0.131.5-40)
- Aleph Zero (`IPFS` apps v0.131.4 and `azero.dev` apps v0.127.2-2)
- Cere Mainnet (`IPFS` apps v0.131.4, but it's working in their `explorer.cere.network`).

Hope this is useful?

Many thanks!!

**_Milos_**

**Comment by jacogr** [comment]
Yes, so that is the weird bit, e.g. see Polkadot (for me) -

![image](https://github.com/polkadot-js/apps/assets/1424473/fe44721e-c037-4788-bfb7-ff1037ea157a)

So, that makes me wonder if it is not the long-standing graph issue. 

Sadly no resolution for that one at all - it is something that people have reported, I cannot replicate and actually occurs inside one of the graph libraries. It is certainly better after some timing work-arounds it would seem, but frustrating as hell since the graph plugin is basically "as is" at this point, aka doesn't get updates.

Previous reports on that - https://github.com/polkadot-js/apps/issues/9567 https://github.com/polkadot-js/apps/issues/8745 https://github.com/polkadot-js/apps/issues/8588

TL;DR Seems to be the same as the long-running graph-plugin-timing-issues - there is nothing that I can think of to try on that anymore. Certainly an issue for some (it would seem), but sadly don't expect any resolution anything soon unless we can drop the plugin or fork it somewhere and address the issues in it.

**Comment by miloskriz** [comment]
Hello @jacogr, 

Many thanks for this response, yes, it looks exactly like these past issues, apologies for reopening as a new one, I am shocked I did not find those when doing my checks...

As it is clearly out of scope of action, I will be closing this instance.

Thanks again!!!...

Best regards

**_Milos_**
